lancero_source.go

```diff
@@ -525,3 +525,27 @@ func (ls *LanceroSource) VoltsPerArb() []float32 {
 	}
 	return v
 }
+
+// SetCoupling set up the trigger broker to connect err->FB, FB->err, or neither
+func (ls *LanceroSource) SetCoupling(status CouplingStatus) error {
```

Comment:
I think this would be more readable with one clause for NoCoupling rather than two if/else, but it looks correct.